### PR TITLE
STREAMLINE-631 Pick runtime jar in libs dir to the default streamline…

### PR DESCRIPTION
**Pull request by HeartSaVioR**
…StormJar config

* Replace `${STREAMLINE_HOME}` in streamlineStormJar to the streamline home directory
  * streamline-server-start.sh finds home directory and pass via Java system property

### CODE REVIEW

File: conf/streamline.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ modules:
     className: com.hortonworks.streamline.streams.service.StreamsModule
     config:
       #change the below to the path on your local machine
-      streamlineStormJar: /tmp/streamline-runtime-storm-0.1.0-SNAPSHOT.jar
+      streamlineStormJar: ${STREAMLINE_HOME}/libs/streamline-runtime-storm-0.1.0-SNAPSHOT.jar
```
Comment (harshach) [inline conf/streamline.yaml]: @HeartSaVioR we could make the config nullable and keep the value in streamline-dev.yaml and take it out of streamliine.yaml and use streamline-dev.yaml for debugging purpose

Comment (HeartSaVioR) [inline conf/streamline.yaml]: @harshach Addressed.

**Comment by HeartSaVioR** [comment]
I did amending previous commit and force pushing, but it seems not reflected to the PR. 
(Maybe changing my repo to public affects here?)

I'll create a new PR and mention you. Thanks.

**Comment by HeartSaVioR** [comment]
Turned out github doesn't recognize my repo as fork of current repo. I'll see how to resolve it.

**Pull request closed without merging**